src/fancy-tools/rectangleRoiTool.js

```diff
@@ -0,0 +1,415 @@
+/* eslint no-loop-func: 0 */ // --> OFF
+import external from './../externalModules.js';
+import baseAnnotationTool from './../base/baseAnnotationTool.js';
+// State
+import { getToolState } from './../stateManagement/toolState.js';
+import toolStyle from './../stateManagement/toolStyle.js';
+import toolColors from './../stateManagement/toolColors.js';
+// Manipulators
+import drawHandles from './../manipulators/drawHandles.js';
+// Drawing
+import { getNewContext, draw, path, setShadow } from './../util/drawing.js';
+import drawLinkedTextBox from './../util/drawLinkedTextBox.js';
+import calculateSUV from '../util/calculateSUV.js';
+
+const cornerstone = external.cornerstone;
+
+const numberWithCommas = (x) => {
```

Comment:
Can we move this to either the `shared/` folder, or to a `utility/` folder? I think the other ROI tools, or anything printing large number, could benefit from this being exposed.

It would also make testing it easier.